reinstall_npm_pkg.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+rm -rf package-lock.json node_modules
+npm install
```

Comment:
Could this be moved to a npm script in package instead?

Reply:
come by mistake. just for my own pleasure